src/platform/packages/shared/deeplinks/security/deep_links.ts

```diff
@@ -29,6 +29,9 @@ export enum SecurityPageName {
   cloudSecurityPostureDashboard = 'cloud_security_posture-dashboard',
   cloudSecurityPostureFindings = 'cloud_security_posture-findings',
   cloudSecurityPostureRules = 'cloud_security_posture-rules',
+  /*
+   * Warning: Computed values are not permitted in an enum with string valued members
```

Comment:
Is this comment necessary here? The value isn't computed?

Reply:
@michaelolo24 I introduced these comments with the Cases IDs at lines 18-19. This is an enum definition and values must be hardcoded, "computed values are not permitted" means that we can not do `cloudDefendPolicies = CLOUD_DEFEND_POLICIES_ID`, so this is just a reminder that the value is not arbitrary, it should match a constant somewhere else (which I miss in this specific comment)

However, if we are deprecating this page, I wonder why we need to keep the `cloudDefendPolicies` ID here.
@CohenIdo is there any reason why we can not clean it?

Reply:
Great! Remember to also remove this comment :)